src/v/coproc/tests/coproc_test_fixture.cc

```diff
@@ -98,36 +78,64 @@ ss::future<coproc_test_fixture::opt_reader_data_t> coproc_test_fixture::drain(
     });
 }
 
+void interpret_batch(
+  std::size_t& n_records_read,
+  model::offset& offset,
+  model::record_batch_reader::data_t& data,
+  model::record_batch_reader::data_t&& batch) {
+    if (batch.empty()) {
+        return;
+    }
+    offset = batch.back().last_offset();
+    for (model::record_batch& r : batch) {
+        n_records_read += r.record_count();
+        data.push_back(std::move(r));
+    }
+}
+
+model::offset next_offset(model::offset o) {
+    if (o == model::model_limits<model::offset>::min()) {
```

Comment:
i think you can now say `model::offset::min()`

Reply:
ok will update in subsequent changes to this file